composer.json

```diff
@@ -29,12 +29,12 @@
         "source": "https://github.com/browscap/browscap-php"
     },
     "require": {
-        "php": ">=7.4.3,<8.3.0",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "^7.5.0",
-        "league/flysystem": "^2.5.0 || ^3.12.0",
-        "matthiasmullie/scrapbook": "^1.4.9",
-        "monolog/monolog": "^2.8.0 || ^3.2.0",
+        "guzzlehttp/guzzle": "^7.8.0",
+        "league/flysystem": "^2.5.0 || ^3.21.0",
+        "matthiasmullie/scrapbook": "^1.5.1",
+        "monolog/monolog": "^2.8.0 || ^3.5.0",
```

Comment:
Is it possible to remove all these duplicate versions and always use the newer version? (e.g always use monolog 3)

Reply:
I think if we remove these, we may have compatibility issues if folks aren't using newer versions of these too. That said, if older deps are being used, the 7.2.* series will still be installable.